hamilton/function_modifiers/recursive.py

```diff
@@ -173,26 +200,10 @@ def __init__(
         self.inputs = inputs if inputs is not None else {}
         self.config = config if config is not None else {}
         self.external_inputs = external_inputs if external_inputs is not None else []
-        self._validate_config_inputs(self.config, self.inputs)
+        _validate_config_inputs(self.config, self.inputs)
```

Comment:
Probably want a try catch here to then contextualize that it's a `subdag` decorator for node `FOO`?

Reply:
Yep good idea